security/expressions.rst

```diff
@@ -61,7 +61,7 @@ Additionally, you have access to a number of functions inside the expression:
     Similar, but not equal to ``IS_AUTHENTICATED_REMEMBERED``, see below.
 ``is_fully_authenticated``
     Similar, but not equal to ``IS_AUTHENTICATED_FULLY``, see below.
-``has_role``
+``is_granted``
```

Comment:
@javiereguiluz I found another one here, not sure if the description is still ok for `is_granted` instead of `has_role`

Reply:
I agree. Here is a quick proposal:

```rst
Checks if the user has the given permission. Optionally accepts a second argument
with the object where permission is checked on. It's equivalent to using
the :doc:`isGranted() method </security/securing_services>` from the authorization
checker service.
```

Reply:
done